libV2/config/index.js

```diff
@@ -271,7 +271,7 @@ class Config {
             warp10Conf.hosts = [{
                 host: _loadFromEnv('WARP10_HOST', config.warp10.host),
                 port: _loadFromEnv('WARP10_PORT', config.warp10.port, _typeCasts.int),
-                nodeId: parsedConfig.nodeId,
+                nodeId: _loadFromEnv('WARP10_NODE_ID', config.warp10.nodeId),
```

Comment:
will it be the node host ?

Reply:
`config.warp10.nodeId`  will be the host warp 10 is on (used for `node` label). The top level `nodeId` is the host the utapi process is running on (used for `origin` label)